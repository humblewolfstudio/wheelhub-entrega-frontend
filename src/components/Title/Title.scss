@import "../../styles/base.scss";

@media(min-width: 700px) {
    .title {
        width: 700px;
        align-self: center;

        .title-content {
            margin: 15px 25px 0px 0px;
        }

        .title {
            font-weight: 900;
            margin-bottom: 40px;
            padding-bottom: 5px;
            position: relative;
        }

        .title::before {
            content: "";
            position: absolute;
            width: 50px;
            left: 0px;
            bottom: 0px;
            border-bottom: 3px solid $green-color;
        }
    }
}

@media(max-width: 700px) {
    .title {
        align-self: flex-start;

        .title-content {
            margin: 15px 25px 0px 0px;
        }

        .title {
            font-weight: 900;
            margin-bottom: 40px;
            padding-bottom: 5px;
            position: relative;
        }

        .title::before {
            content: "";
            position: absolute;
            width: 50px;
            left: 0px;
            bottom: 0px;
            border-bottom: 3px solid $green-color;
        }
    }
}