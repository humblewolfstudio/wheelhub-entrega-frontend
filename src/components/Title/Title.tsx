import React from 'react';
import './Title.scss';

export default function Title() {
    return (
        <div className="title">
            <div className="title-content">
                <h4 className='title'>Test Frontend Wheel Hub</h4>
            </div>
        </div>
    )
}