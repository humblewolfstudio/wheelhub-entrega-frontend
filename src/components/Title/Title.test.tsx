import React from 'react';
import * as renderer from 'react-test-renderer';
import Title from './Title.tsx';

it('title showing correctly', () => {
    const component = renderer.create(
        <Title />
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})