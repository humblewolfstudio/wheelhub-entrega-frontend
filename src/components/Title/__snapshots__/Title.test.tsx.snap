// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title showing correctly 1`] = `
<div
  className="title"
>
  <div
    className="title-content"
  >
    <h4
      className="title"
    >
      Test Frontend Wheel Hub
    </h4>
  </div>
</div>
`;
